### ISSUE TITLE
Readthedocs is outdated

### ISSUE THREAD

**Issue by oxfn** [neon-jungle/wagtailnews#21]
Hello! Thank you for this nice module
It would be great to actualise https://wagtail-news.readthedocs.io/en/latest/setup.html
I see that docs in repo are correct

**Comment by JOJ0** [comment]
I agree, this was very misleading and confusing when initially configuring this wagtail extension.
Thanks for a very useful extension and all your efforts though!!! Appreciated.
